apps/server/src/modules/board/uc/column.uc.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { BoardDoAuthorizable, BoardRoles, ContentElementType, UserRoleEnum } from '@shared/domain';
+import { setupEntities, userFactory } from '@shared/testing';
+import { cardFactory, columnBoardFactory, columnFactory } from '@shared/testing/factory/domainobject';
+import { LegacyLogger } from '@src/core/logger';
+import { AuthorizationService } from '@modules/authorization';
+import { ContentElementService, BoardDoAuthorizableService, CardService, ColumnService } from '../service';
+import { ColumnUc } from './column.uc';
+
+describe(ColumnUc.name, () => {
```

Comment:
Create a test set for students (even when they are not allowed to perform any operation)
Create a test set for invalid users